pandas_gbq/gbq.py

```diff
@@ -146,12 +148,17 @@ def __init__(self, project_id, reauth=False, verbose=False,
         _test_google_api_imports()
         self.project_id = project_id
         self.reauth = reauth
-        self.verbose = verbose
         self.private_key = private_key
         self.dialect = dialect
         self.credentials = self.get_credentials()
         self.service = self.get_service()
 
+        if verbose is not None:
+                warnings.warn(
+                    "verbose is deprecated and will be removed in "
```

Comment:
Could we add a unit test to exercise the deprecation warning? See `test_generate_bq_schema_deprecated` for similar example